partition/grub.go

```diff
@@ -81,3 +81,7 @@ func (g *grub) SetBootVars(values map[string]string) error {
 	}
 	return env.Save()
 }
+
+func (g *grub) RebootForUpdate(afterMins int) error {
+	return reboot(afterMins)
```

Comment:
This (and the one for uboot) are untested currently and this will reduce the test-coverage. Not a big deal, we can do a followup branch but if we can get a test for this cheaply that would be great.